core/templates/dev/head/css/oppia.css

```diff
@@ -2873,3 +2873,514 @@ md-card.preview-conversation-skin-supplemental-card {
   color: red;
   margin-bottom: 10px;
 }
+
+.conversation-skin-progress-dots {
```

Comment:
Oh, I meant including them in conversation_skin directive, as we did before. That would work for now (and as a side effect would make it easier to see the diff.)